apps/src/bounce/bounce.js

```diff
@@ -1204,6 +1205,9 @@ Bounce.setBackgroundImage = function (backgroundUrl) {
 };
 
 Bounce.loadTiles = function (tiles, goalTiles) {
+  if (!skin.drawTiles) {
```

Comment:
Football, basketball, and soccer just have a simple background image, and don't need the red wall or separate goal tile images.